core/src/main/java/org/apache/iceberg/actions/BaseRewriteDataFilesAction.java

```diff
@@ -262,12 +264,16 @@ public RewriteDataFilesActionResult execute() {
     return tasksGroupedByPartition.asMap();
   }
 
-  private void replaceDataFiles(Iterable<DataFile> deletedDataFiles, Iterable<DataFile> addedDataFiles) {
+  @VisibleForTesting
+  void replaceDataFiles(Iterable<DataFile> deletedDataFiles, Iterable<DataFile> addedDataFiles) {
     try {
       RewriteFiles rewriteFiles = table.newRewrite();
       rewriteFiles.rewriteFiles(Sets.newHashSet(deletedDataFiles), Sets.newHashSet(addedDataFiles));
       commit(rewriteFiles);
-    } catch (Exception e) {
+    }  catch (CommitStateUnknownException e) {
+      LOG.warn("Failed to check commit status: skipping data file deletion.", e);
+      throw e;
+    }  catch (Exception e) {
```

Comment:
Looks like formatting is incorrect here: there are two spaces between `}` and `catch` for both clauses and that is causing git to detect that the catch statement for `Exception` has changed when it actually hasn't. Can you fix those, please? We try to minimize changes that can cause git conflicts.